src/handlers/createHandler.tsx

```diff
@@ -327,12 +319,9 @@ export default function createHandler<
 
       if (Platform.OS === 'web') {
         // typecast due to dynamic resolution, attachGestureHandler should have web version signature in this branch
-        (
-          RNGestureHandlerModule.attachGestureHandler as AttachGestureHandlerWeb
-        )(
+        (RNGestureHandlerModule as any).attachGestureHandlerWeb(
```

Comment:
In that case I believe it will be the best to [ad ts-ignore](https://github.com/software-mansion/react-native-gesture-handler/pull/2873/commits/85a03ba00a4eb8c456b8cfefac04d89e0f043f12). The problem here is that type of Gesture Handler Module comes from `Spec` which extends `TurboModule` and I don't want to change native implementation.

Reply:
That's better, but I'd like to avoid duplicating the definitions if possible. Have you tried adding the 4th optional argument to the `attachGestureHandler`? It should make it work without casting but I'm not sure how will it play with codegen.

You'd likely need to make some tricks to make both TS and codegen happy at the same time, but this should work:

```jsx
type UnsafeMixed = unknown;

...
  attachGestureHandler: (
    handlerTag: number,
    newView: number,
    actionType: number,
    propsRef?: UnsafeMixed
  ) => void;
...
```

You would need to also make sure the native code still works, and likely update the generated specs we're using on the old arch.